src/types/index.ts

```diff
@@ -0,0 +1,25 @@
+export interface DeviceError extends Error {
+  message: string;
+}
+
+/*
+ * TODO: These are types that should be coming from unchained-bitcoin
+ */
+
+// this one COULD come from unchained-bitcoin except the relevant
+// file isn't typescript and so the conversions don't seem to work
+export type BitcoinNetwork = "mainnet" | "testnet" | "signet" | "regtest";
+
+export interface KeyDerivation {
+  bip32Path: string;
+  fingerprint: string;
+}
+
+export interface TxInput {
+  // this one is messy in the way it's used
+  // and so we'll need to better define it as typescript
+  // conversion gets propagated
+  multisig: unknown;
+  transactionHex: string;
+  index: number;
+}
```

Comment:
Added this as an explanation to the README here: [117210c](https://github.com/unchained-capital/unchained-wallets/pull/87/commits/117210c4f9d4f8f4243dc809e7200f15ffacebba)

In short, first I think you're thinking of a `type.ts` or `types.d.ts` file. That doesn't work in this project (yet) because of the `isolatedModules` setting which is needed for division of labor with babel. 

Also, for clarification, it's not a redefinition, but rather defining things that aren't there yet because unchained-bitcoin is only partially typescript-ified and so pulling the types from there was causing issues. Also didn't want to slow down velocity with another PR there, but we should do that soon.